build.sc

```diff
@@ -131,6 +131,9 @@ object Deps {
   val lambdaTest = ivy"de.tototec:de.tobiasroeser.lambdatest:0.7.1"
   val log4j2Core = ivy"org.apache.logging.log4j:log4j-core:2.18.0"
   val osLib = ivy"com.lihaoyi::os-lib:0.8.1"
+  val millModuledefsVersion = "0.10.9-alpha-1"
```

Comment:
Why are we using an alpha version?

Reply:
It's until we settle on this setup. There will be a stable release (`0.10.9`) once we do. 

Currently (before this PR), Mill also contains a `mill-moduledefs` artifact, and we need to distinguish it from the other one. Hopefully, once we tag Mill 0.10.9, we can also release it as `0.10.9` without any collisions. 

Also, Maven-compatible tools treat `0.10.9-alpha-1` as _before_ `0.10.9`, and the meaning of `alpha` in software development will make it semantically clear. Although in Mill, we used to treat a `0.10.9-1` as _after_ `0.10.9`.